object-manipulation-2.tar/object-manipulation-2/src/defaults.js

```diff
@@ -0,0 +1,4 @@
+function defaults(target,source){
+  const newObject = Object.assign(target, source);
```

Comment:
instead of using `Object.assign`, try looping over the keys in the `source`, and if the `key` doesn't exist on the `target` object, assign the key/value pair from the source to the target object (ie. `target[key] = source[key]`)